settings.gradle

```diff
@@ -78,8 +78,10 @@ apply from: "${rootDir}/gradle/scripts/settings-flags.gradle"
 
 // Published BOM projects
 includeWithFlags ':bom',                                 'bom'
+project(':bom').projectDir.mkdirs()
 // Published version catalog projects
 includeWithFlags ':version-catalog',                     'version-catalog'
+project(':version-catalog').projectDir.mkdirs()
```

Comment:
- Should we create these projects under `gradle/virtual-projects`?
- Add `gradle/virtual-projects/` to `.gitignore`?

Reply:
Maybe `gradle/scripts/.virtual-projects/`? I wonder if it makes sense to use `build/virtual-projects` as well.. :thinking:

Reply:
`build/virtual-projects` sounds good because `build/` is already excluded from git and is automatically cleaned.

Reply:
Sounds good.  I'll try to move generated projects under `build/virtual-projects`.

Reply:
Done :)
Please take a look